R/print.R

```diff
@@ -42,7 +42,7 @@ print.gsplot <- function(x, ...){
 
   top.par <- par(no.readonly=T)
   for (i in which(names(views) %in% 'view')){
-    par(top.par)
+    # par(top.par)
```

Comment:
Wrote this in the wrong place. I think it's because the layout function knows how to define par$usr